### DISCUSSION TITLE
重複記事、コピー記事の対策

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#186] [要望・アイデア・不具合]
Contributions稼ぎかは判らないですが、既存記事の丸写しやコンパイルしただけの記事を投稿し続けるユーザーが居ました。

https://qiita.com/ohisama@github/items/e3a45b5cee8ce2b6d689
https://qiita.com/ohisama@github/items/f4d5c0634396d3321090
https://qiita.com/ohisama@github/items/d8db755be82ad683ebe0
https://qiita.com/ohisama@github/items/ef4ae9008d68aa1030b2
https://qiita.com/ohisama@github/items/ce02748e394ab574dbb1

自分がよく見るC#タグだけでこれだけ出てきたので、多分もっとありそうな気がしますが、自分が無視するだけならミュート＆ブロックでいいのですが、こういったものがまかり通ると記事の質の低下と既存記事の価値を下げる事に繋がると思いますので、何らかの対策をして欲しいです。完全な対策は難しいかもしれませんが、

- 記事の連続投稿間隔制限（但し下書きはOK）
- 一定数通報があったら自動的に記事を非表示にする。通報の多いユーザーは、前述の連続投稿間隔制限を重くする。
- 投稿時に同タグで似たような本文・タイトルを含む記事を抽出し、あれば確認画面を表示する（難しいかな？）
- Contributions計算方法の見直し

辺りはあった方がいい気がしますが、いかがでしょう？

**Comment by shiracamus** [comment]
以前、私の記事もパクられたことがありましたが、著作権侵害であることを運営に伝えて対処していただきました。

それとは毛色が違いますが・・・
本日、ある方から大量にLGTMされました。
おそらく自動化していると思われるのですが、こういった行為もなんとかしたいですね。

あと、GitHubランキングやセキュリティニュースまとめを自動投稿している方もいて、自動投稿に関するガイドラインもあるといいですね。他所情報のパクリと言えそうな気もするので。

**Comment by ohakutsu** [comment]
@radian-jp 
ご意見ありがとうございます！

実は、ご提示いただいたうちのいくつかはやっていたりします。

- 記事の連続投稿間隔制限
    - スパム対策として連続投稿に制限をかけています
- Contributions計算方法の見直し
    - 現在、Contributionの計算内容を変更検討しています
    - https://twitter.com/Qiita/status/1506452456796221443

ただ、現状以上の制限を自動的にかけるとすると、誤判定が多くなってしまう可能性があり、難しいと考えています。

根本的な解決ができず申し訳ないのですが、コミュニティガイドラインの違反であれば、通報をもとに対応を行っていますので、ご不便をおかけしますが通報をしていただけると幸いです :pray:
https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0

**Reply by radian-jp** [reply]
通報はしておりますが、上記ユーザーのコピー記事が増え続けていますね。完全に野放し状態では？
2022年03月25日で3回、2022年03月27日で4回投稿されており、制限が掛かってるとはとても言えないです。真面目に自分で考えて書いた記事なら、一日に何回も投稿出来るものではないと思います。
タグ汚染されるのも迷惑なので、もう既にミュート＆ブロックしましたが、明らかなスパムではないが、記事の体裁を成していないものについては、通報が集まりにくいかもしれません。

**Reply by kskwtnk** [reply]
@radian-jp 
ご意見ありがとうございます！

申し訳ありません、感情としては非常に共感をしているのですが……

> 一日に何回も投稿出来るものではない

例えば私自身、筆が乗っていれば1日に3記事投稿することもあります
だからというわけではないのですが、1日1~2記事までと制限するのも若干強すぎる縛りではないか、と考えています

> 完全に野放し状態

こちらについては表現が難しいのですが、できるだけ正直にお答えします

まず`記事単位の通報`については一定数の通報が集まり次第投稿者に警告を出しています
（[ヘルプの記事](https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0)にて記載している通りのフローを実施しています）
逆に言えば、明らかな利用規約違反の記事でなければ、運営から強制的な対応を実施することはありません
強制的に非公開などにした場合、多くの方に不安感や不信感を与えかねないため、現在はこういった対応をとっています

そして`ユーザー単位の対応`については、更に慎重にならざるを得ないのが実情です
運営の介入による行動の規制やアカウントの凍結などを実施すると、記事単位よりも更に大きなハレーションを生んでしまいます

---

ここまで言い訳めいたことを書き連ねて参りましたが、運営としても何かを丸写しした記事がQiitaに増えて欲しくはありません
ただ、いわゆる「良い記事」の比率を高めるにしても、コミュニティのみなさんが気持ち良く受け入れられるよう実行するのは非常に難しく感じています

運営が強制的に介入する量を減らしつつ上手く記事や執筆者が巡っていくバランスを探っている最中でありまして、ご容赦いただけますと幸いです 🙇
